package.json

```diff
@@ -40,7 +40,7 @@
     "clang:check": "yarn clang-format --dry-run --Werror addon/**",
     "clang:fix": "yarn clang-format -i addon/**",
     "cleanup:binaries": "rm -rf build prebuilds duckdb",
-    "download-duckdb": "rm -rf duckdb && curl -L https://github.com/cwida/duckdb/archive/7c1a2f2867dfe5b02717e7bd7cf7ed46fbe08a34.tar.gz > duckdb.tar.gz && tar xf duckdb.tar.gz && mv duckdb-7c1a2f2867dfe5b02717e7bd7cf7ed46fbe08a34 duckdb && rm duckdb.tar.gz",
+    "download-duckdb": "rm -rf duckdb && curl -L https://github.com/cwida/duckdb/archive/6d3586322e55390fd5d9280febaf4d53fdc927c0.tar.gz > duckdb.tar.gz && tar xf duckdb.tar.gz && mv duckdb-6d3586322e55390fd5d9280febaf4d53fdc927c0 duckdb && rm duckdb.tar.gz",
```

Comment:
you also need to increment the package.json version of `node-duckdb` so that the binaries are rebuilt in ci/cd (atm it's using the current version's prebuilt binaries)